pkg/hakeeper/checkers/dnservice/check.go

```diff
@@ -50,6 +51,7 @@ func Check(
 	currTick uint64,
 ) []*operator.Operator {
 	stores, reportedShards := parseDnState(cfg, dnState, currTick)
+	fmt.Println("check parameters:", dnState, *reportedShards)
```

Comment:
No more fmt.Println -- always use logger.

Reply:
This is for testing...
This pr has some bug...